functions.gallery.php

```diff
@@ -35,7 +44,12 @@ function wp_enqueue_media() {
 
 	/**
 	 * Outputs a view template which can be used with wp.media.template
-	 */
+	 *
```

Comment:
Something like "Filter the default gallery type." could be a good title for that filter.